Formula/quickjs.rb

```diff
@@ -25,3 +35,39 @@ def install
     assert_equal "hello", output
   end
 end
+
+__END__
```

Comment:
Has this patch been reported and accepted upstream?

Reply:
It's in progress. Basically the author could not get the dynamically loaded JS modules to compile in OSX so he disabled this if environment is OSX. I found a solution and removed those conditions (full functionality of the Makefile is now enabled under OSX).

In my opinion even if it does not provide any additional functionality OOTB it proves that it works. I think it'd be a good idea to separately provide various bindings as dynamically loadable binary JS modules - this would allow to decouple them from the core quickjs and they wouldn't have to be compiled-in. I've got a plan to use this feature.

Reply:
Basically this patch along with `make test` ensures that the resulting binaries are capable of loading binary JS modules built as *shared libraries* under OSX.

Reply:
I have removed this from build.